pkg/wamr/Makefile

```diff
@@ -1,10 +1,10 @@
 PKG_NAME=wamr
 PKG_URL=https://github.com/bytecodealliance/wasm-micro-runtime.git
-ifeq ($(PKG_BLEADING),1)
+ifeq ($(PKG_BLEEDING),1)
   PKG_VERSION=main
```

Comment:
If this doesn't work, you can just drop it.

Reply:
does work for me ( just tested)
```
$ cd <RIOT>/examples/wasm
$ PKG_BLEADING=1 make 
$ cd  <RIOT>/build/pkg/wamr 
$ git status 
On branch main
Your branch is up to date with 'origin/main'.
```

```
$ cd <RIOT>/examples/wasm
$ make 
$ cd  <RIOT>/build/pkg/wamr 
$ git status 
HEAD detached at dc4dcc3
```

Reply:
Thank you for catching the typo